PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your pull request.
+
+- [ ] Please add `[WIP]` to the beginning of your pull request title if you are still working on it or it is not ready for review.
+- [ ] Please add `[MRG]` to the beginning of your pull request title if all checks have passed and there are no merge conflicts with the base branch. For each reviewer that approves your pull request, add `+num_approvals` to the title (e.g. `[MRG+1]`, `[MRG+2]`, etc.).
+- [ ] Please verify that your code satisfies the [code/documentation quality guidelines](http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines).
+- [ ] Documentation and high-coverage tests are necessary for enhancements to be accepted, but you may wish to provide evidence for the benefit of your enhancement with distinguishing examples and benchmarks.
```

Comment:
Benchmarks are often shown in the PR discussion, while examples are in the code. Please separate this comment from that about documentation and tests in the code.